### DISCUSSION TITLE
Crystallizer Property Models

### DISCUSSION THREAD

**Discussion by AARON683** [watertap-org/watertap#1180] [General]
Hello everyone, 

I am currently designed a model using the crystallizer block. Am I correct in saying that the cryst_prop_pack is the only property model you can use for this unit block as it contains the relevant solubility and density correlations? If this is true, then currently NaCl is the only salt solution that can be treated within the crystallizer model? or is there other property models which you can input into the crystallizer model for other salts, for example, calcium and magnesium salts??

**Comment by OOAmusat** [comment]
Hi @AARON683,

Thank you for your interest in using the crystallizer model.

The ``cryst_prop_pack`` is the only property model that can be used with the crystallizer at the moment, and NaCl is the only salt solution that can be treated. The model is however easily extensible to other salts if the parameters for the properties are available.

We have some ongoing work on extending the crystallizer model to Mg and Ca salts, but they're still under development.

**Reply by AARON683** [reply]
Thank you @OOAmusat for confirming this, I look forward to the Mg and Ca salt property models being released!
